.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Report a problem with concrete
+title: ''
+labels: triage required
```

Comment:
Do we need to create the label first ? Is that the proper format when the label has a space inside ? In the github documentation I see:
```
labels: [bug, triage]
```

Reply:
If the label does not exists, it does nothing, if it exists it is automatically added. So yes label must be created.